src/pages/stay-secure.js

```diff
@@ -0,0 +1,177 @@
+var React = require('react');
+var Footer = require('../components/footer.js');
+var ContentContainer = require('../components/content-container.js');
+import {Accordion, Tab} from '../components/mozilla-accordion.js';
+import ga from 'react-ga';
+import { browserHistory } from 'react-router';
+import StickContainer from '../components/sticky-container.js';
+
+module.exports = React.createClass({
+  onButtonClick: function() {
+    window.location.href = '/stay-secure/thank-you/?action=user_updated';
+  },
+  getPosition: function() {
+    if (!this.refs.stickyContainer) {
+      return 0;
+    }
+    return this.refs.stickyContainer.getClientRects()[0].y + 50 + window.scrollY - window.innerHeight;
```

Comment:
Because the sticky part of the container can change for mobile based on which expanded accordion items are open, we need to calculate where the sticky point is.